bin/codemods/README.md

```diff
@@ -67,6 +67,24 @@ If you're developing your own transformations, it may be useful to know you can
 ./node_modules/.bin/jscodeshift -t transformation.js [target files]
 ```
 
+## How to debug codemods
+
+If you are a codemod author, you may want to debug your codemod using the Chrome debugger. Then
+run the codemod script with a `--debugger` parameter:
+```bash
+./bin/codemods/run.js --debugger my-transform client/target.js
+```
+or
+```bash
+npm run codemod -- --debugger my-transform client/target.js
+```
+This will run `jscodeshift` in a Node process that has command line arguments `--inspect` and
+`--debug-brk`. That means that a debugger will be attached to the Node process and will break on
+the first statement. That allows you to connect with Chrome and run the codemod script.
```

Comment:
It sounds like we may be changing the node args soon. How about leaving off the specifics:

> This will run `jscodeshift` in a Node process with the appropriate flags for debugging. The process will break on the first `debugger` statement, allowing inspection with Chrome.